libraries/libfc/CMakeLists.txt

```diff
@@ -78,11 +78,9 @@ if(APPLE)
   add_library(zstd INTERFACE)
 endif()
 
-find_package(Boost 1.66 REQUIRED COMPONENTS
-    date_time
-    chrono
-    unit_test_framework
-    iostreams)
+if(NOT boost_headers_SOURCE_DIR)
+   find_package(Boost REQUIRED COMPONENTS date_time chrono unit_test_framework iostreams)
+endif()
```

Comment:
since fc isn't an external submodule any longer (unlike appbase/chainbase), this seems unnecessary

Reply:
Yes, right, removed.